addons/point_of_sale/models/point_of_sale_config.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import uuid
+
+from openerp import models, fields, api, _
```

Comment:
can you please arrange ??????
![alphabet-coloring-pages-printable-apple-and-bird](https://cloud.githubusercontent.com/assets/14940179/12125932/81349dd4-b3ab-11e5-859f-d1c45a678d30.gif)